src/rpm/headers/types.rs

```diff
@@ -15,17 +15,17 @@ pub struct RPMPackageSegmentOffsets {
 
 /// Describes a file present in the rpm file.
 pub struct RPMFileEntry {
-    pub(crate) size: u64,
-    pub(crate) mode: FileMode,
-    pub(crate) modified_at: u32,
-    pub(crate) sha_checksum: String,
-    pub(crate) link: String,
-    pub(crate) flags: FileFlags,
-    pub(crate) user: String,
-    pub(crate) group: String,
-    pub(crate) base_name: String,
+    pub size: u64,
+    pub mode: FileMode,
+    pub modified_at: u32,
+    pub sha_checksum: String,
+    pub link: String,
+    pub flags: FileFlags,
+    pub user: String,
+    pub group: String,
+    pub base_name: String,
     #[allow(unused)]
-    pub(crate) dir: String,
+    pub dir: String,
     pub(crate) content: Vec<u8>,
```

Comment:
`get_file_entries()` is pretty pointless if none of the fields are accessible (and there are no accessors)

Reply:
Could you remove the allow unused annotation too?

Reply:
done